releasenotes/notes/logs-container-configuration-toggle-f5f8b2831c6a575f.yaml

```diff
@@ -0,0 +1,16 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
```

Comment:
There is a better section for breaking changes: `upgrade`. These appear first in the release notes, and include all the things to pay attention to when upgrading to a newer version of the agent.

Reply:
fixed